custom/icds_core/view_utils.py

```diff
@@ -30,15 +30,15 @@ def is_icds_cas_project(domain):
 
 def check_authorization(domain, user, master_app_id):
     if (
-        IS_ICDS_ENVIRONMENT
-        and toggles.ROLE_WEBAPPS_PERMISSIONS.enabled(domain)
+        (IS_ICDS_ENVIRONMENT or IS_ICDS_STAGING_ENVIRONMENT)
```

Comment:
Can't you remove the environment checks in the followup PR?

Reply:
Ohh!! I had this question and it had slipped my mind. Thank you for pointing out.
Do we need the restriction for ICDS envs only or we want to have it for all the envs? Because if we will restrict it to ICDS then these changes would not affect any env in any way. 
Otherwise, there would be a prerequisite to creating roles and assigning them to users in other environments?